.ci/run_tests.sh

```diff
@@ -19,4 +19,6 @@ run_oldtests
 
 install_nvim
 
+run_single_includes_test
```

Comment:
When I was writing this I was not yet sure that if additional executables were built they will not be installed, so wrote test after install test. Though I now know that they are not installed and it should be possible to put this call there as well.